game-headless/scripts/run_bot

```diff
@@ -76,7 +76,7 @@ readonly LOBBY_HOST=lobby.triplea-game.org
 # The default value is the port for the TripleA community's public lobby.
 # Under normal circumstances, you will not have to change this variable.
 #
-readonly LOBBY_PORT=3304
+readonly LOBBY_HTTPS_PORT=8080
```

Comment:
80 is typical for servers that serve HTML, 8080 is typical for applicaton servers, which also sometimes do use 443. I think more when `Lobby_port` is dead, `Lobby_https_port` will be renamed to `lobby_port` and we can get rid of this anti-pattern of baking in the technology name with the variables. Basically I really regrest the https part of this name, it's similar IMO to having: `JAVA_NIO_SOCKET_LOBBY_TCP_PORT`.

On another level, using non-standard ports does reduce the number of port-scans and attacks against a server. Because we do not need to interface with standard port numbers, there would be some value there (eg: port 9000 or something else that is not standard).

Reply:
Agreed, but I don't think choosing a non-standard port helps at all.
Search engines like https://shodan.io crawl the whole web for those kind of things